packages/image-ui/src/ImageElement/ImageElement.tsx

```diff
@@ -64,50 +81,53 @@ export const ImageElement = (props: ImageElementProps) => {
       <div contentEditable={false}>
         <figure css={styles.figure?.css} className={`group ${styles.figure?.className}`}>
           <div css={styles.optionsToolbar.css} className={styles.optionsToolbar.className}>
-            <button
-              css={styles.optionsToolbarButton?.css}
-              className={styles.optionsToolbarButton?.className}
-              onClick={() => setSize(ImageSizeType.large)}>
-              lg
-            </button>
-            <button
-              css={styles.optionsToolbarButton?.css}
-              className={styles.optionsToolbarButton?.className}
-              onClick={() => setSize(ImageSizeType.medium)}>
-              md
-            </button>
-            <button
-              css={styles.optionsToolbarButton?.css}
-              className={styles.optionsToolbarButton?.className}
-              onClick={() => setSize(ImageSizeType.small)}>
-              sm
-            </button>
+            <ImageSizeButton
```

Comment:
Since screen and lg are the same width we need to see the currently selected image size. Maybe underline or highlight it with color.